oeis/217/A217589.asm

```diff
@@ -1,8 +1,14 @@
 ; A217589: Bit reversed 16-bit numbers.
-; Submitted by Simon Strandgaard
+; Coded manually 2022-02-01 by Simon Strandgaard, https://github.com/neoneye
```

Comment:
Great! Can you please also add A217589 to `oeis/protect.txt` to avoid that it gets overwritten?